xds/src/main/java/io/grpc/xds/XdsClusterResource.java

```diff
@@ -382,6 +382,11 @@ static boolean hasNegativeValues(Duration duration) {
     return upstreamTlsContext;
   }
 
+  private static boolean isSystemRootCertsEnvVarSet() {
+    String useSystemRootCerts = System.getenv("GRPC_EXPERIMENTAL_XDS_SYSTEM_ROOT_CERTS");
+    return useSystemRootCerts != null && useSystemRootCerts.equals("true");
```

Comment:
`Boolean.parseBoolean(useSystemRootCerts)` is cleaner. But I think Larry has started using `GrpcUtil.getFlag()` now, which also looks for a system property.

Reply:
Using mutable (public) static field that is edited by the tests now.